ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/Backgrounds/NewTabPageBackgroundButtonsView.swift

```diff
@@ -142,12 +151,34 @@ class NewTabPageBackgroundButtonsView: UIView, PreferencesObserver {
         $0.centerX.equalToSuperview()
       }
     }
+
+    playButton.snp.remakeConstraints {
+      $0.centerX.equalToSuperview()
+      $0.centerY.equalToSuperview().offset(20)
+    }
+    // Hide the play button if the video is in landscape mode on a phone
+    if isLandscape && UIDevice.isPhone {
+      playButton.isHidden = true
+    } else {
+      playButton.isHidden = !playButtonAdded
+    }
```

Comment:
Lets shift logic like this to functions such as `updatePlayButtonVisibility` so that business logic exists in a single location reducing the chance of race conditions on whether a button should or should not be shown